app/components/markdown/markdown.tsx

```diff
@@ -520,6 +524,19 @@ const Markdown = ({
         );
     };
 
+    const renderMaxNodesWarning = () => {
+        const styles = [baseTextStyle, style.maxNodesWarning];
+
+        return (
+            <FormattedText
```

Comment:
We're potentially going to make this component look a bit nicer in a future PR, but I'm hoping most people don't hit this, so  it's shippable in the current state